code-challenges.test.js

```diff
@@ -40,11 +80,30 @@ const letterE = "e"
 
 // b) Create the function that makes the test pass.
 
+const letterFilter = (array, letter) => {
+  return array.filter(array => array.includes(letter))
+}
+console.log(letterFilter(arrayOfWords1, letterA))
+console.log(letterFilter(arrayOfWords2, letterE));
```

Comment:
It is great to use console.logs to work through your coding process. But this file should be run by your jest test. I would imagine if you ran the file you would get back errors in the current form.